simulation/amaru-sim/src/simulator/simulate.rs

```diff
@@ -262,11 +291,105 @@ mod tests {
         assert_eq!(world.run_world(), &Vec::new());
     }
 
-    // NOTE: This will be replaced with a test that uses an in-process NodeHandle later, which
-    // avoids the problem of locating and spawning the binary.
+    #[test]
+    #[should_panic]
+    fn simulate_pure_stage_echo() {
+        let config = Default::default();
```

Comment:
Please don’t use `Default::default()`: the inferred type is unavailable here on github. There is no downside to using `WhateverType::default()` instead.